client/src/data/topics.ts

```diff
@@ -235,6 +32,5 @@ export const getTopicOptions = (motivation: MotivationType) =>
       value: topic.value,
     }));
 
-export const topics = topicsData.sort((a, b) => a.value - b.value);
-
-export const shuffleTopics = shuffleArray(topicsData);
+export const shuffleTopics = shuffleArray(await getTopics());
```

Comment:
Don't use top-level awaits because it may not be supported in some old browsers and hence the checks on this PR are failing.
Try to come up with something that does not involve top-level awaits.